indico/web/client/js/custom_elements/ind_time_picker.js

```diff
@@ -0,0 +1,8 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2025 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import CustomElementBase from 'indico/custom_elements/_base';
```

Comment:
This file is pretty much empty at the moment 🙈

Reply:
Ah crap. Yeah, I was gonna rename ind_combo_box to ind_time_picker and move time-picker-related logic there. Cause we're not really using the combobox for what it was designed for. I can still do this in another PR if you want.